app/javascript/components/map/MapStatistics.vue

```diff
@@ -105,36 +108,59 @@ export default {
     description: String,
     tabs: Array,
     layers: Array,
-    oecmLayer: Object,
     oecmPresent: Boolean,
     tilesUrl: String,
+    tilesUrlOecm: String,
     disclaimer: Object,
   },
 
   data() {
     return {
+      activeTabId: '',
+      includeOecms: false,
       map: {},
+      mapLoaded: false,
       isActive: true,
+      layersActive: [],
+      layersDefault: [],
+      layersOecm: [],
       mapboxToken: process.env.MAPBOX_TOKEN,
-      allLayers: [],
+      tabsActive: [],
+      tabsDefault: [],
```

Comment:
I'm not sure `tabsDefault` is needed as you assign `this.tabs` as its value in `created` and from then on use it in the rest of the script. Perhaps you might be able to just replace `tabsDefault` with `this.tabs` wherever it's used.